ci-operator/jobs/infra-image-mirroring-multi-arch.yaml

```diff
@@ -1,4 +1,3 @@
-# DO NOT EDIT. This file is auto-generated.
```

Comment:
Was this intentional?

Reply:
No.  I have no idea why this file changed

Reply:
rebased. repushed